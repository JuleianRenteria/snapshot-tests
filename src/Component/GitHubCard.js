import Card from 'react-bootstrap/Card'
import  PretendMe  from './PretendMe.webp'

function GitHubCard () {
    return (
        <Card style={{width: '18rem'}}>
            <Card.Img variant='top ' src='{PretendMe}' alt='Famouse Female Scientist' />
                <Card.Body>
                    <Card.Title> Juleian Renteria </Card.Title>
                    <Card.Subtitle className='mb-2 text-muted'> Starting Up Software Developer </Card.Subtitle>
                    <Card.Text>
                        Hi! My name is Juleian Renteria and I am pursuing a career in Software 
                        Development. I am a father of 2 and plan to be married once my Coding career takes off!
                    </Card.Text>
                    <Card.Link href='https://github.com/GiaMadonna1'> Find me on Github Here! </Card.Link>
                </Card.Body>
        </Card>
    );
}

export default GitHubCard