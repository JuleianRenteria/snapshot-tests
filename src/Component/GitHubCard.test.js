import GitHubCard from "./GitHubCard";
import renderer from 'react-test-renderer'
import { render } from "@testing-library/react";


test('renders a snapshot', () => {
    const tree = renderer.create(<GitHubCard />).toJSON()
    expect(tree).toMatchSnapshot()
})