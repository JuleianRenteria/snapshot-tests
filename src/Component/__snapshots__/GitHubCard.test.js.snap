// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    alt="Famous Male Lawyer"
    className="card-img-top "
    src="{PretendMe}"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
       Juleian Renteria 
    </div>
    <div
      className="mb-2 text-muted card-subtitle h6"
    >
       Starting Up Software Developer 
    </div>
    <p
      className="card-text"
    >
      Hi! My name is Juleian Renteria and I am pursuing a career in Software 
      Development. I am a father of 2 and plan to be married once my Coding career takes off!
    </p>
    <a
      className="card-link"
      href="https://github.com/JuleianRenteria"
    >
       Find me on Github Here! 
    </a>
  </div>
</div>
`;